eZ/Publish/Core/FieldType/BinaryBase/Type.php

```diff
@@ -148,11 +148,11 @@ protected function completeValue(BaseValue $value)
     }
 
     /**
-     * BinaryBase does not support sorting.
+     * BinaryBase does not support sorting, yet.
      *
      * @param \eZ\Publish\Core\FieldType\BinaryBase\Value $value
      *
-     * @return bool
+     * @return mixed
```

Comment:
Note: this will cause trouble in the future. Here though it's out of scope (something for 8.x).